ax/models/torch/botorch_modular/sebo.py

```diff
@@ -233,16 +232,34 @@ def optimize(
             candidates, a tensor with the associated acquisition values, and a tensor
             with the weight for each candidate.
         """
+        # Prepare arguments for optimizer
+        optimizer_options_with_defaults = optimizer_argparse(
+            self.acqf,
+            optimizer_options=optimizer_options,
+            optimizer="optimize_acqf_homotopy",
+        )
+
+        _tensorize = partial(torch.tensor, dtype=self.dtype, device=self.device)
+        ssd = search_space_digest
+        bounds = _tensorize(ssd.bounds).t()
+
+        optimizer_options["batch_initial_conditions"] = get_batch_initial_conditions(
```

Comment:
Should this be set on `optimizer_options_with_defaults`? We now construct `optimizer_options_with_defaults` but don't pass it down to the optimizers.

I think this will also address the pyre failure:
fbcode/ax/models/torch/botorch_modular/sebo.py:246:8 Undefined attribute [16]: Optional type has no attribute `__setitem__`.

```suggestion
        optimizer_options_with_defaults["batch_initial_conditions"] = get_batch_initial_conditions(
```

Reply:
For this is looks like there is a pyre pre-commit so will look at adding it to the pre-commit hooks https://github.com/facebook/pyre-check